### ISSUE TITLE
Suggestion Effects

### ISSUE THREAD

**Issue by CupidonSauce173** [genboy/Festival#56]
You should add that when a player enter in a certain area, it will add a effect on him.

**Comment by genboy** [comment]
Yep, I was thinking about that 2 butaAt the moment the effect flag will remove all effects on players in the area, this will also remove the fire-protection effect. It might be possible to use a command on-enter-event to give the player(@p) a fire-protection effect.
Not to have fire burning when there is no damage to the player was a first important step, next i'll have to test the fire is also extinguished on enter area where no damage to the player is possible.

**Comment by CupidonSauce173** [comment]
By effects, I was talking about these one (for example) : Jump boost, Speed, invisibility, blindness, nausea, etc..It would be really nice

**Comment by genboy** [comment]
@CupidonSauce173 
:) Yes, this plugin is made to do this with commands attachted to an area event, this way any command (unknown plugins and future pmmp commands) can be used.

Example:
/fe command  arename action cmdid commandstring

areaname = (your area name), 
action = enter, center or leave (to add the command on the specific event), 
cmdid = (command id ), 
commandstring = effect {player} speed 120 10 (or what ever you like commanding :)

**Comment by CupidonSauce173** [comment]
@genboy can I feel dumb ?

*facepalm*

**Comment by genboy** [comment]
The possibilities are infinite I believe, we will all get lost somehow :)
